project.clj

```diff
@@ -1,10 +1,10 @@
-(defproject saml20-clj "0.1.6"
```

Comment:
How did we release this as `kirasystems/saml20-clj` before, without having `kirasytems` in the `project.clj`? Either way, probably shouldn't have the `k2n` prefix here.